addon/webextension/content-script.js

```diff
@@ -54,11 +59,18 @@ function findAndReplace(wordList) {
     <br/><a href="${SUPPORTURL}" target="_blank", rel="noopener noreferrer">
     [return to blissful ignorance]
     </a>`;
-    hoverEle.classList.add("donotdelete-tooltip");
-    hoverEle.setAttribute("data-tooltip-position", "right");
-    node.appendChild(hoverEle);
   });
 
+  // between 1-5 seconds, flip them back, but keep the over.  see #22
```

Comment:
Timing is always very hard. For example, unless the "Privacy" text appears "above the fold", it may be unlikely that the user will even see the effect. 2-6 seconds may not be enough time to read the content and scroll down to observe the effect if it's in a "Privacy Policy" footer link. 🤷‍♀️